content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,73 @@
+===================================
+Manufacturing product configuration
+===================================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In order to manufacture a product in Odoo *Manufacturing*, the product must be properly configured.
+Doing so consists of enabling the Manufacturing route and configuring a Bill of Materials (BoM) for
```

Comment:
formatting suggestions
- italicize the route name since it's a clickable setting
- lowercase the term since it's not a proper noun, and bolden the first instance of it like we might do in a textbook (I think I'd like that to be a team-wide guideline, wdyt?)

```suggestion
Doing so consists of enabling the *Manufacturing* route and configuring a **bill of materials (BoM)** for
```